### PR TITLE
SPDX license tags and seL4 Foundation setup

### PR DESCRIPTION
**Pull request by lsf37**

### CODE REVIEW

File: default.xml [hunk 1]
```diff
@@ -1,21 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-     Copyright 2020, Data61
```
Comment (kent-mcleod) [inline default.xml:3]: This license header is generated by the CI during the publish phase.  That script may need updating to generate an SPDX header if it hasn't already.

**Review by kent-mcleod — APPROVED** [review]

File: default.xml (hunk 1 above)
Comment (kent-mcleod) [reply default.xml:3]: (the same applies to every default.xml in all the -manifest repos.)

Comment (lsf37) [reply default.xml:3]: Yeah, you're right. I have a PR for that on bitbucket ready to go, but holding off until all of these are merged. We already have a race condition in one of the manifests, but it'll go back to SPDX once I merge the CI change, so it should be fine.

**Comment by lsf37** [comment]
@ssrg-bamboo `test`

**Pull request merged**